src/main/webapp/app/overview/course-conversations/layout/conversation-selection-sidebar/conversation-selection-sidebar.component.html

```diff
@@ -76,6 +76,60 @@ <h3 class="card-title">
                             </ng-template>
                         </jhi-conversation-sidebar-section>
                     </div>
+                    <!-- Exercise Channels -->
+                    <div [hidden]="searchTerm && searchTerm.length > 0 && !displayedExerciseChannelConversations?.length" class="mb-2" id="exercise-channel-section">
```

Comment:
This is now more or less duplicated five times. Could you create a component for that and provide the different values as input?